docs/development/airflow.md

```diff
@@ -1,65 +1,67 @@
 # Airflow workflow
 
-The next section describes how to run Airflow workflows locally while performing computation in Google Cloud Platform. This is useful for testing and debugging, but for production use, we recommend running Airflow on a dedicated server.
+This section describes how to set up a local Airflow server which will orchestrate running workflows in Google Cloud Platform. This is useful for testing and debugging, but for production use, it is recommended to run Airflow on a dedicated server.
 
-## Pre-requisites
+## Install pre-requisites
 
 - [Docker](https://docs.docker.com/get-docker/)
 - [Google Cloud SDK](https://cloud.google.com/sdk/docs/install)
 
 !!!warning macOS Docker memory allocation
-    If you are working on a macOS, the default amount of memory available for Docker might not be enough to get Airflow up and running. You should allocate at least 4GB of memory for the Docker Engine (ideally 8GB). [More info](https://airflow.apache.org/docs/apache-airflow/stable/howto/docker-compose/index.html#)
+    On macOS, the default amount of memory available for Docker might not be enough to get Airflow up and running. Allocate at least 4GB of memory for the Docker Engine (ideally 8GB). [More info](https://airflow.apache.org/docs/apache-airflow/stable/howto/docker-compose/index.html#)
 
 
 ## Configure Airflow access to Google Cloud Platform
 
-Run the next command with the appropriate <PROJECT_ID> to ensure you have Google default application credentials set up:
+!!!warning Specifying Google Cloud parameters
+    Run the next two command with the appropriate project ID and service account name to ensure the correct Google default application credentials are set up.
+
+Authenticate to Google Cloud:
 
 ```bash
-gcloud auth application-default login --project=<PROJECT_ID>
+gcloud auth application-default login --project=open-targets-eu-dev
```

Comment:
if we want to hardcode a project in the documentation, we should be using `open-targets-genetics-dev`

Reply:
Actually, you know what, you're right. I only did this for consistency, because the service account (open-targets-genetics-dev@appspot.gserviceaccount.com) was specified, and project ID wasn't.

However, a better way to achieve consistency is to not hardcode anything in the first place. So this is what I did now. Fixed in https://github.com/opentargets/genetics_etl_python/pull/221/commits/d42fd268f781ef31aaf56911349995d7c64615d1.